File: app/src/main/java/org/mifos/mobilebanking/api/local/SelfServiceDatabase.java
package org.mifos.mobilebanking.api.local;

import com.raizlabs.android.dbflow.annotation.Database;

@Database(name = SelfServiceDatabase.NAME, version = SelfServiceDatabase.VERSION)
public class SelfServiceDatabase {

    public static final String NAME = "SelfService"; // we will add the .db extension

    public static final int VERSION = 1;
}

File: app/src/main/java/org/mifos/mobile/api/local/SelfServiceDatabase.kt
package org.mifos.mobile.api.local

import com.raizlabs.android.dbflow.annotation.Database

@Database(name = SelfServiceDatabase.NAME, version = SelfServiceDatabase.VERSION)
object SelfServiceDatabase {
    const val NAME: String = "SelfService" // we will add the .db extension
    const val VERSION: Int = 1
}
